### DISCUSSION TITLE
Beginner - Deployment error with website does not load the theme and all links are broken

### DISCUSSION THREAD

**Discussion by khang2107** [alshedivat/al-folio#2313] [Q&A] [answered]
I just started creating my personal website using al-folio repo with jekyll theme.

I followed the instruction in the install.md. Here are the steps I did:
1. I forked the al-folio to my github repo and renamed it to khang2107.github.io
2. I cloned it to my local machine.
3. In _config.yml, I changed the URL to [https://khang2107.github.io/](url).
4. In _pages/about.md, I added "This is a test".
5. I staged, committed, and pushed.
6. I went to the settings, changed branch into gh-pages.

I waited for the actions to load. However, there are some errors in deployment.

I tried deleting the repo and followed the instruction again with the new repo under https://khang2107.github.io/ but still not worked.

The errors that I am having include:
1. The website does not display correctly. Even though the details from about.md is updated, the **overall layout is not right**.
2. I have tried refreshing the page and resync with the GitHub multiple times but the website is still not displaying correctly.
3. All the links are broken when I access.

Here is the website link: 
[https://khang2107.github.io/](url)

Here is the link to my repo:
[https://github.com/khang2107/khang2107.github.io](url)

Please if anyone knows how to fix this, please help me. I have spent 4 hours straight to try to fix the problems but still could not find it.

I really appreciate your help.

<img width="1452" alt="image" src="https://github.com/alshedivat/al-folio/assets/65332993/fafdb975-7fe4-4125-9204-d5e57a13ba97">
-
<img width="1312" alt="image" src="https://github.com/alshedivat/al-folio/assets/65332993/76306a80-59d0-43c2-bd8c-935773798ba9">
-
<img width="1153" alt="image" src="https://github.com/alshedivat/al-folio/assets/65332993/d8976068-d27a-4224-b919-60d01703f875">
-
<img width="1457" alt="image" src="https://github.com/alshedivat/al-folio/assets/65332993/70d6883c-13c8-406f-8249-de61ff56959e">
-
<img width="1065" alt="image" src="https://github.com/alshedivat/al-folio/assets/65332993/1f704224-9a5e-40a6-bb69-81442c106e9b">

**Comment by george-gca** [comment, marked as answer]
Follow [INSTALL.md](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#recommended-approach) recommended approach to the letter.

You forgot to do this step 3 part: `leave baseurl empty (do NOT delete it).`

**Reply by lilylovewine** [reply]
Hi George, i faced the same issue like Khang. I also followed your inutructuction but my theme was totally off like Khang. https://lilylovewine.github.io/le.vinergy/. Do you  know how to fix this error?

![theme off](https://github.com/user-attachments/assets/d21283ab-8b56-430e-97c8-91c6a6e3501b)

**Reply by lilylovewine** [reply]
![Lighthouse Badger](https://github.com/user-attachments/assets/287619a9-3ef2-41c7-a104-1d7e0503cd17)

**Reply by lilylovewine** [reply]
![_config](https://github.com/user-attachments/assets/dce3f9f8-e572-4ff9-b336-304ec44c31de)

**Reply by george-gca** [reply]
In your case, the issue is with your repository name. If you check our [INSTALL.md](https://github.com/alshedivat/al-folio/blob/main/INSTALL.md#recommended-approach) first step:

> If you plan to upload your site to `<your-github-username>.github.io`, note that the name of your repository :warning: **MUST BE** :warning: `<your-github-username>.github.io` or `<your-github-orgname>.github.io`, as stated in the [GitHub pages docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites).

Your repository name is `le.vinergy`, which is wrong.

Now, if you want your site based on our template to have this url: https://lilylovewine.github.io/le.vinergy/

You can keep your repository with the same name, but set this in your `_config.yml`:

```yml
url: https://lilylovewine.github.io
baseurl: /le.vinergy
```

**Comment by khang2107** [comment]
I found it, I thought I should have not removed anything for the baseurl but now I removed it and it worked. Thank you so much.
